tests/decompile-test/baselines/tagged_templates.blocks

```diff
@@ -15,12 +15,18 @@
 <value name="VALUE">
 <shadow type="math_number"><field name="NUM">0</field></shadow>
 <block type="spritescreate">
-<field name="img">img&#38;&#35;96&#59;&#38;&#35;10&#59;0&#38;&#35;10&#59;&#38;&#35;10&#59;&#38;&#35;10&#59;1&#38;&#35;10&#59;2&#38;&#35;10&#59;3&#38;&#35;96&#59;</field>
+<field name="img">img`
```

Comment:
this doesn't look right... shouldn't newlines get escaped?

Reply:
My understanding is that they shouldn't need to be escaped in xml like that re: https://www.w3.org/TR/xml-c14n/#Example-WhitespaceInContent 

I believe this example would break on decompiling the image if it weren't being read correctly, right? As the https://arcade.makecode.com/app/0610dda95754bed7e6df6ce7485f7d128dd0f4ea-1375337f35#pub:_Rd0UJmiP0RWq

can always add a few more values to be escaped if we're worried about it, would probably be better to just use https://www.npmjs.com/package/he if we want the full solution though as the current escaping is broken -- e.g. it's causing crashes due to malformed encodings here https://github.com/microsoft/pxt-arcade/issues/2774

Reply:
iiirc we had bugs around that before. can't remember if it was us or blockly